doc/source/whatsnew/v0.25.0.rst

```diff
@@ -172,7 +172,7 @@ The repr now looks like this:
 
 .. ipython:: python
 
-    from pandas.io.json import json_normalize
+    from pandas import json_normalize
```

Comment:
But this is causing the doc build failure, because there is a warning being thrown.

Reply:
Ah makes sense. Can you convert this into a `code-block` directive instead of `ipython` then? You should see examples of that in old whatsnew files that compare before / after behavior of changes